devito/mpi/distributed.py

```diff
@@ -308,29 +308,33 @@ def glb_to_rank(self, index):
 
         Parameters
         ----------
-        index : int or list of ints
-            The index, or list of indices, for which the owning MPI rank(s) is
+        index : array of ints
+            The index, or array of indices, for which the owning MPI rank(s) is
             retrieved.
         """
-        assert isinstance(index, (tuple, list))
-        if len(index) == 0:
+        assert isinstance(index, np.ndarray)
```

Comment:
Maybe this fixes the python 3.9 issue as well?

Reply:
possibly, where is the issue for it again?

Reply:
I think no issue was opened, need to check